### PR TITLE
Change the generated projects default branch name to main

### PR DESCRIPTION
**Pull request by SudharakaP**
This renames the default branch of the generated project to main

Resolves https://github.com/jhipster/jhipster-online/issues/258

### CODE REVIEW

File: src/main/java/io/github/jhipster/online/service/GitService.java [hunk 1]
```diff
@@ -70,6 +70,8 @@ public void pushNewApplicationToGit(User user, File workingDir, String organizat
         remoteAddCommand.setUri(urIish);
         remoteAddCommand.call();
 
+        String currentBranch = git.getRepository().getFullBranch();
+        if (currentBranch.equals("refs/heads/master")) git.branchRename().setNewName("main").call();
```
Comment (pascalgrimaud) [inline src/main/java/io/github/jhipster/online/service/GitService.java:74]:
```suggestion
        if (currentBranch.equals("refs/heads/master")) {
            git.branchRename().setNewName("main").call();
        }
```

Comment (pascalgrimaud) [reply src/main/java/io/github/jhipster/online/service/GitService.java:74]: don't we have prettier:format for Java ?

**Comment by pascalgrimaud** [comment]
LGTM and trusting you, with really minor comment, feel free to go ahead :)

File: src/main/java/io/github/jhipster/online/service/GitService.java (hunk 1 above)
Comment (SudharakaP) [reply src/main/java/io/github/jhipster/online/service/GitService.java]: We do, but in our CI we haven't enforced it; I'll enable it in a separate PR. 😄

**Comment by SudharakaP** [comment]
> LGTM and trusting you, with really minor comment, feel free to go ahead :)

Thanks. I'll integrate the prettier to the CI soon. That will help I think with our prettier problems. 👍🏽

**Pull request merged**